spec/controllers/tasks_controller_spec.rb

```diff
@@ -4,14 +4,31 @@
 RSpec.describe TasksController, type: :controller do
 
     describe "POST#create" do
-        it" returns http success " do
+        it "returns http success " do
             post :create, params: { task: {:description => "test1"}}
             expect(response).to have_http_status(:success)
         end
 
-        it" returns an error if the task is not valid " do
+        it "returns an error if the task is not valid " do
             post :create, params: { task: {:description => nil}}
             expect(JSON.parse(response.body)).to eq("error" => "The task couldn't be added. Try again!")
         end
     end 
+
+    describe "GET#index" do
+        it "returns http success " do
+            get :index
+            expect(response).to have_http_status(:success)
+        end
+
+        it "returns body with description attribute" do
+            get :index
+            expect(JSON.parse(response.body)[0].keys).to include("description")
+        end
+
+        it "returns body with is_completed attribute" do
+            get :index
+            expect(JSON.parse(response.body)[0].keys).to include("is_completed")
+        end
```

Comment:
These two tests are failing. Can you see why? 👀

Reply:
I am not sure but I suppose is because it will make real request and if there is no tasks in the database then the tests will fail.